dotCMS/src/integration-test/java/com/dotcms/rest/BundleResourceTest.java

```
@@ -223,4 +241,88 @@ public void test_downloadBundle_success() throws DotDataException {
+        createFilter();
+
+        //Generate bundle file
+        bundle.setOperation(PushPublisherConfig.Operation.PUBLISH.ordinal());
+        final File bundleFile = APILocator.getBundleAPI().generateTarGzipBundleFile(bundle);
+
+        PublisherAPIImpl.class.cast(APILocator.getPublisherAPI()).getFilterDescriptorMap().clear();
+
+        //Call download endpoint
+        final Response responseResource = bundleResource.downloadBundle(getHttpRequest(),response,bundleId);
+        Assert.assertEquals(Status.OK.getStatusCode(),responseResource.getStatus());
+
+        assertTrue(bundleFile.exists());
+
+        final Folder systemFolder = APILocator.getFolderAPI().findSystemFolder();
+        final String bundleDir = bundleFile
+                .getAbsolutePath()
+                .substring(0, bundleFile.getAbsolutePath().indexOf(".tar.gz"));
+        recursiveFindAndRun(
+                Paths.get(bundleDir),
+                path -> {
+                    Logger.info(BundleResourceTest.class, "OJO:>> " + path.toFile().getAbsolutePath());
+                    assertNotEquals(("/system host" + HOST_EXTENSION), path.toFile().getName());
+                    assertNotEquals((systemFolder.getIdentifier() + FOLDER_EXTENSION), path.toFile().getName());
+                });
+    }
+
+    private static void recursiveFindAndRun(final Path findInPath, final Consumer<Path> action) {
+        try (DirectoryStream<Path> newDirectoryStream = Files.newDirectoryStream(findInPath)) {
+            StreamSupport
+                    .stream(newDirectoryStream.spliterator(), false)
+                    .peek(path -> {
+                        action.accept(path);
+                        if (path.toFile().isDirectory()) {
+                            recursiveFindAndRun(path, action);
+                        }
+                    })
+                    .collect(Collectors.toList());
+        } catch (IOException e) {}
```

Codacy found an issue: [Avoid empty catch blocks](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6393381)